### DISCUSSION TITLE
I feel punished for investing and burning resources in the game

### DISCUSSION THREAD

**Discussion by Chucrutty** [sunflower-land/sunflower-land#2686] [General]
![image](https://github.com/sunflower-land/sunflower-land/assets/125780525/c679b5f5-53eb-4695-ab5b-4fb3067f2633)

I've been cooking a lot and I don't save resources, and I don't feel I have any advantage next to the players who are only withdrawing resources, I would like to be able to expand and continue to grow in lvl, I've been cooking for some time without any stimulus, but I like the game and a lot of you. That's why I keep investing and cooking non-stop, I'd like to be chasing the game, not the game chasing me

**Comment by future2back** [comment]
Friend, expansions and new levels are on the horizon. However, I don't share the same sentiment as you do. I believe that this kind of strategy keeps the player much more prepared than others in many ways. And let's not forget that creating a vast divide for new players isn't healthy for the game in the long run.

**Reply by Chucrutty** [reply]
As I am the player who burns the most resources and invests in the game, I would like some progress ....

**Comment by dragonhunter2017** [comment]
:) Don't be upset, the new expansion will come and it will very hard to reach lvl 60 and beyond for most of us except you :)

**Reply by Chucrutty** [reply]
problem is wanting to invest in the game and not being able to, being willing to spend 200 gold + and continue expanding, but in this case the game does not allow me
